### PR TITLE
Turn conda builds into no-ops

### PR DESCRIPTION
**Pull request by atalman**
Same as: https://github.com/pytorch/test-infra/pull/5917/
Remove builds and tests

Related Issue
https://github.com/pytorch/pytorch/issues/138506

Test:
https://github.com/pytorch/test-infra/actions/runs/11900158899/job/33160415240?pr=5933

### CODE REVIEW

**Review by kit1980 — APPROVED** [review]

**Comment by malfet** [comment]
Why only 4 checks were run before merging? Previously I've made change to the workflow 13 checks (including `Test build linux conda` were run), see https://github.com/pytorch/test-infra/pull/5917/checks /  https://github.com/pytorch/test-infra/actions/runs/11829271648/job/32960808268?pr=5917

File: .github/workflows/build_conda_linux.yml [hunk 1]
```diff
@@ -84,9 +84,7 @@ jobs:
       REF: ${{ inputs.ref }}
       CU_VERSION: ${{ matrix.desired_cuda }}
     name: ${{ matrix.build_name }}
-    runs-on: ${{ matrix.validation_runner }}
-    container:
-      image: alpine:3.14
```
Comment (malfet) [inline .github/workflows/build_conda_linux.yml:89]: Why those two lines were deleted? It will results in slower checkout for no-op jobs

Comment (atalman) [reply .github/workflows/build_conda_linux.yml:89]: We don't need to run-on validation_runner which maybe GPU machine

**Comment by malfet** [comment]
Oh, I see you've removed the test workflows, which IMO is wrong as those workflows are still being invoked, so imo it should have been done in the following 3 steps:
 - Change all of them to no-ops
 - Slim down the matrix
 - Finally delete the tests (perhaps a month later or something)

File: .github/workflows/build_conda_linux.yml (hunk 1 above)
Comment (atalman) [reply .github/workflows/build_conda_linux.yml:89]: Here is the test for:
https://github.com/pytorch/vision/actions/runs/11891587028/job/33160971728

**Comment by atalman** [comment]
Why we need to keep tests around if they are not being used ?

**Comment by malfet** [comment]
> Why we need to keep tests around if they are not being used ?

If something is unused, we should delete the workflow

**Pull request merged**